threat-model/Threat-Model-Draft-2022-08-09.md

```diff
@@ -0,0 +1,240 @@
+# Security Considerations
+
+
+## Document Status: DRAFT
+
+This document is currently a **draft**, submitted to the PATCG (and eventually the PATWG) with the intention of receiving broad feedback. In this state, it does not yet represent any form of consensus in either group, nor should it be seen as a specific endorsement from any contributors. This document will likely evolve significantly through the consensus process.
+
+
+## Introduction
+
+In this document, we outline the security considerations for proposed purpose-constrained APIs for the web platform (that is, within browsers, mobileOSs, and other user-agents) specified by the Private Advertising Technologies Working Group (PATWG).
+
+We assume that an active attacker can control the network and has the ability to corrupt any number of clients, the parties who call the proposed APIs, and some (proposal specific) subset of aggregators.
+
+In the presence of this adversary, APIs should aim to achieve the following goals:
+
+
+
+1. **Privacy**: Clients (and, more specifically, the vendors who distribute the clients) trust that (within the threat models), the API is purpose constrained. That is, all parties learn nothing beyond the intended result (e.g., a differentially private aggregation function computed over the client inputs.)
+2. **Correctness:** Parties receiving the intended result trust that the protocol is executed correctly. Moreover, the amount that a result can be skewed by malicious input is bounded and known.
+
+Specific proposed purpose constrained APIs will provide their own analysis about how they achieve these properties.
+
+
+## 1. Threat Model
```

Comment:
Maintaining these section numbers will be a real pain.  Is this a consequence of the conversion process?

Reply:
I just exported markdown from Google docs. if there is tooling that would help, happy to migrate.

Reply:
I've used https://word2md.com/ in the past.  I don't know if that adds numbers, but maybe this is because the source document had numbers in titles.  I don't know how you managed that either.